cmd/cosign/cli/sign.go

```diff
@@ -223,12 +226,13 @@ func SignCmd(ctx context.Context, keyPath string,
 			}
 		}
 	}
-	index, err := cosign.UploadTLog(signature, payload, pemBytes)
+	inclusionProof, uuid, err := cosign.UploadTLog(signature, payload, pemBytes)
 	if err != nil {
 		return err
 	}
-	fmt.Println("tlog entry created with index: ", index)
-	return nil
+	uploadOpts.RekorInclusionProof = inclusionProof
```

Comment:
Should this only be uploaded when bundling is specified?

Reply:
Yes! We check for that [here](https://github.com/sigstore/cosign/pull/224/files#diff-52fed6da2e81889fa98c519123e97900541f69ec5ad0d490b164c684da5b51feR211-R213)

Reply:
Ahhh I guess it comes down to network access vs space in the image if they're being experimental